scripts/ClearTestResults.ps1

```diff
@@ -21,3 +21,6 @@ RmRF -DirectoryName $testResultsDirName
 foreach ($srcSubdir in (Get-ChildItem "src")) {
     RmRF -DirectoryName "$srcSubdir\$testResultsDirName"
 }
+
+RmRF -DirectoryName $env:Temp -Recurse
```

Comment:
the reason for this one is that many of our tests create temporary files in this directory and don't clean them up after execution. I for one every so often reach a disk limit and at that point tests fail with an exception from the filesystem